tests/testthat/test-shinytest2-modules.R

```diff
@@ -72,27 +72,6 @@ testthat::test_that("e2e: filter panel shows all the datasets when datanames is
   app$stop()
 })
 
-testthat::test_that("e2e: filter panel is not displayed when datanames is NULL", {
-  # TODO - this is not satisfied
```

Comment:
This is now satisfied but covered in other tests in `test-shinytest2-reporter.R`